CK3ToEU4Tests/Copy_Files.bat

```diff
@@ -0,0 +1,5 @@
+echo on
+rem Imagemagick dlls
+xcopy "..\imageMagick\dll-windows-required" "Release" /Y /E /I
+mkdir "Release\TestFiles"
+xcopy "TestFiles" "Release\TestFiles" /Y /E /I
```

Comment:
For test files, I've gotten in the habit of setting them up in VS and CMakelists instead of in a copy script. Not a big deal for a handful of dds files, but it might be useful for habit building and thus enabling some tricks for other test files in the future.